src/main/kotlin/com/github/hanseter/json/editor/util/CustomizationObject.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.hanseter.json.editor.util
+
+import com.github.hanseter.json.editor.types.TypeModel
+
+/**
+ * An object that can be used to customize the behavior and appearance of the editor.
+ */
+interface CustomizationObject {
+
+    /**
+     * Gets the title for a given element.
+     */
+    fun getTitle(model: TypeModel<*, *>, defaultTitle: String): String {
```

Comment:
It might make sense to also add customization options for the description now and add more options later on.